jwst/regtest/test_miri_caldector1.py

```diff
@@ -0,0 +1,93 @@
+from glob import glob
+import os
+
+import pytest
+from astropy.io.fits.diff import FITSDiff
+from jwst.pipeline import Detector1Pipeline
+from jwst.pipeline.collect_pipeline_cfgs import collect_pipeline_cfgs
+from jwst.stpipe import Step
+
+
+@pytest.mark.bigdata
+def test_miri_caldetector1_rate(_jail, rtdata, fitsdiff_default_kwargs):
+    rtdata.get_data("miri/image/jw00001001001_01101_00001_MIRIMAGE_uncal.fits")
+
+    collect_pipeline_cfgs("config")
+    args = ["config/calwebb_detector1.cfg", rtdata.input]
+    Step.from_cmdline(args)
+    rtdata.output = "jw00001001001_01101_00001_MIRIMAGE_rate.fits"
+
+    rtdata.get_truth("truth/test_miri_caldetector1/jw00001001001_01101_00001_MIRIMAGE_rate.fits")
+    assert rtdata.output != rtdata.truth
+
+    diff = FITSDiff(rtdata.output, rtdata.truth, **fitsdiff_default_kwargs)
+    assert diff.identical, diff.report()
+
+
+@pytest.fixture(scope="module")
+def run_pipeline(jail, rtdata_module):
+    """Run calwebb_detector1 pipeline on MIRI imaging data."""
+    rtdata = rtdata_module
+    rtdata.get_data("miri/image/jw00001001001_01101_00001_MIRIMAGE_uncal.fits")
+
+    step = Detector1Pipeline()
+    step.lastframe.save_results = True
+    step.firstframe.save_results = True
+    step.dq_init.save_results = True
+    step.saturation.save_results = True
+    step.rscd.save_results = True
+    step.linearity.save_results = True
+    step.dark_current.save_results = True
+    step.refpix.save_results = True
+    step.jump.save_results = True
```

Comment:
Technically the `jump` step product should be identical to the `ramp` product that you get as a result of setting `save_calibrated_ramp=True`, so you don't really need both of them.